cherrypy/test/test_static.py

```diff
@@ -215,13 +215,13 @@ def test_config_errors(self):
         self.assertErrorPage(500)
         if sys.version_info >= (3, 3):
             errmsg = (
-                'TypeError: staticdir\(\) missing 2 '
+                r'TypeError: staticdir\(\) missing 2 '
```

Comment:
The argument of `assertMatchesBody()` is a regular expression. `(` and `)` have special meaning in regular expressions (grouping) and need to be escaped for matching literal `(` and `)`.